lib/plugins/pre_commit/checks/jscs.rb

```diff
@@ -17,8 +17,8 @@ def self.description
       def call(staged_files)
         return 'JSCS executable could not be located' if jscs_source.nil?
 
-        # Check for .js files
-        staged_files = staged_files.grep(/\.js$/)
+        # Check for .js files NOT in the Webpack Pipeline
+        staged_files = staged_files.grep(/.+(?!packs).+\.js$/)
```

Comment:
do these files always end in `packs` or `_packs`? If so, we shouldn't have the `.+` after the capture group because it could match unintended things (like `unpacks.js` or somethjing)